content/en/blog/2023/basic-connector/index.md

```diff
@@ -0,0 +1,574 @@
+---
+title: Basic Connector
+linkTitle: How to Create Connector # Mandatory, make sure that your short title.
+date: 2023-07-11 # Put the current date, we will keep the date updated until your PR is merged
+author: 'Gordon Hamilton' # If you have only one author, then add the single name on this line in quotes.
+
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+spelling:
+  cSpell:ignore Spammetrics Servicegraph exampleconnector struct Errorf
+  cSpell:ignore mapstructure pdata tailtracer Mapstructure pmetric ptrace uber
```

Comment:
I would stick to a single casing for mapstructure (I think lowercase makes more sense)